src/Desktop/Resources/ErrorMessages.Designer.cs

```diff
@@ -19,7 +19,7 @@ namespace Axe.Windows.Desktop.Resources {
     // class via a tool like ResGen or Visual Studio.
     // To add or remove a member, edit your .ResX file then rerun ResGen
     // with the /str option, or rebuild your VS project.
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "17.0.0.0")]
```

Comment:
I noticed this change happened after I edited the resx file, should I revert or is it an expected change?